<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create3DGroupDialog</class>
 <widget class="QDialog" name="Create3DGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create 3D Group</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter a name for the 3D group and select signals for X, Y, Z axes:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Group Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtGroupName">
       <property name="placeholderText">
        <string>e.g. Position Plot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>X Axis Signal:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbXSignal"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Y Axis Signal:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbYSignal"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Z Axis Signal:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmbZSignal"/>
     </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_attitude">
      <property name="text">
       <string>Attitude:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cmbAttitude"/>
    </item>
    <!-- Euler selectors -->
    <item row="5" column="0">
     <widget class="QLabel" name="label_roll">
      <property name="text">
       <string>Roll Signal:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cmbRollSignal"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_pitch">
      <property name="text">
       <string>Pitch Signal:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="cmbPitchSignal"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_yaw">
      <property name="text">
       <string>Yaw Signal:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="cmbYawSignal"/>
    </item>
    <!-- Quaternion selectors -->
    <item row="8" column="0">
     <widget class="QLabel" name="label_qx">
      <property name="text">
       <string>Qx Signal:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="cmbQxSignal"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_qy">
      <property name="text">
       <string>Qy Signal:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="cmbQySignal"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_qz">
      <property name="text">
       <string>Qz Signal:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="cmbQzSignal"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_qw">
      <property name="text">
       <string>Qw Signal:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QComboBox" name="cmbQwSignal"/>
    </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>